plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -28,6 +28,37 @@ const {
 	onboardingUrl,
 } = window.wcWcpaySubscriptions;
 
+const ErrorNotice = ( { isError }: { isError: boolean } ) => {
+	if ( ! isError ) {
+		return null;
+	}
+
+	return (
+		<Notice
+			className="wcpay-empty-subscriptions__error"
+			status="error"
+			isDismissible={ false }
+		>
+			{ createInterpolateElement(
+				__(
+					'WooCommerce Payments failed to install. To continue with WooCommerce Payments built-in subscriptions functionality, please install <a>WooCommerce Payments</a> manually.',
+					'woocommerce-payments'
+				),
+				{
+					a: (
+						// eslint-disable-next-line jsx-a11y/anchor-has-content
+						<a
+							href="https://woocommerce.com/payments"
```

Comment:
Should we link to .org instead? i.e. [https://wordpress.org/plugins/woocommerce-payments/](https://wordpress.org/plugins/woocommerce-payments/)

My reason for suggesting this is that there's quite a bit of friction involved when attempting to download a free plugin from woocommerce.com (especially if you're not logged in).

i.e. when you attempt to download WCPay from woocommerce.com you are taken here:

<img width="1160" alt="Screen Shot 2022-05-03 at 2 29 29 pm" src="https://user-images.githubusercontent.com/1527164/166404954-4e6cb46d-a8db-420e-b9ba-324f47419c36.png">

When you click the "Proceed to checkout" button you are taken here:

<img width="794" alt="Screen Shot 2022-05-03 at 2 29 42 pm" src="https://user-images.githubusercontent.com/1527164/166404992-bea8be7b-93a8-4fea-8aec-0590337699ea.png">

IMHO this is a lot to ask to download a free plugin and might lower conversions.

Reply:
Is there a way to show the WCPay plugin in the `/wp-admin/plugin-install.php` admin page?
Of so, would this be suitable?

Reply:
I've switched the link to point to https://wordpress.org/plugins/woocommerce-payments/ as suggested ([6b7fd14](https://github.com/woocommerce/woocommerce/pull/32852/commits/6b7fd14c0975d73b9b510eea4fb5688a736320c5)).

> Is there a way to show the WCPay plugin in the /wp-admin/plugin-install.php admin page?

I don't believe so. When the `'DISALLOW_FILE_MODS'` constant is present, the wp-admin/plugin-install.php page becomes inaccessible. 

<img width="1077" alt="Screen Shot 2022-05-03 at 4 23 20 pm" src="https://user-images.githubusercontent.com/8490476/166412076-84ffbaad-1243-434a-8257-666c11cd8009.png">